connector_jira_tempo/migrations/12.0.1.0.0/pre-migrate.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2019 Brainbean Apps (https://brainbeanapps.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from openupgradelib import openupgrade
```

Comment:
FTR you don't need openupgradelib for this ;)

Reply:
I'm no expert there, will premigrate script work just like that?

Reply:
yes, please find existing folders `migrations` in other modules. Eg:

connector/connector/migrations/12.0.1.1.0/pre-migration.py
server-auth/auth_api_key/migrations/12.0.2.0.0/post-migrate.py

Reply:
Done! Can we merge now? :D